docs/CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+
+## PLACEHOLDER FOR CELL TYPING RELEASE DATE
+
+* Downloads will now contain cell type annotations as inferred with [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html) and [`CellAssign`](https://github.com/Irrationone/cellassign).
```

Comment:
```suggestion
* Cell type annotations are now included in each download. 
Cells were annotated using both [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html) and [`CellAssign`](https://github.com/Irrationone/cellassign).
```
Maybe something like this? I just don't know how I feel about use of "inferred".